Automattic/simplenote-ios
# Nü passcode screen: VI

### Fix

This PR improves Voice Over support on Passcode screens.

* When allowed passcode can be dismissed using "escape" gesture
* Error messages are now announced without changing focus from the keyboard
* Use haptic feedback when error occurs

Related issue: #761

### Test (Setting a passcode)
1. Enable VoiceOver
2. Open a screen to set up passcode

- [x] Check that when screen header changes, for example after transition to the confirmation screen, it is announced
- [x] Screen can be dismissed using "escape" gesture (draw "z" with two fingers)

### Test (Turning a passcode off)
1. Enable VoiceOver
2. Setup a passcode
3. Open a screen to turn off passcode

- [x] When entering incorrect passcode, the error message is announced
- [x] When entering incorrect passcode there is haptic feedback

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
